connectors/connectors_cli.py

```diff
@@ -213,108 +222,114 @@ async def ensure_index_exists():
 
 
 async def create_connectors_index():
+=======
+# @TODO print help page when no arguments passed
+@click.group(invoke_without_command=True)
+@click.option('-v', '--version', is_flag=True, callback=print_version,
+              expose_value=False, is_eager=True)
+def cli():
+    pass
+    # ctx = click.get_current_context()
+    # click.echo(ctx.get_help())
+    # ctx.exit()
 
-# async def _start_service(actions, config, loop):
-#     """Starts the service.
+@click.command(help="Authenticate Connectors CLI with an Elasticsearch instance")
+def login():
+    click.echo('login command')
 
-#     Steps:
-#     - performs a preflight check using `PreflightCheck`
-#     - instantiates a `MultiService` instance and runs its `run` async function
+cli.add_command(login)
+
+# Connector group
+@click.group(invoke_without_command=True)
+def connector():
+    click.echo('test connector')
+
+cli.add_command(connector)
+
+
+# Index group
+@click.group(invoke_without_command=True)
+def index():
+    click.echo('testing index')
+>>>>>>> Stashed changes
+
+cli.add_command(index)
+
+# Job group
+@click.group(invoke_without_command=True)
+def job():
+    click.echo('testing job')
+
+cli.add_command(job)
+
+
+# def run(args):
```

Comment:
Maybe it makes sense to remove these comments completely? It's really hard to read and review the code with it

Reply:
Doing it step by step. There are some parts I will re-use in the feature so I want to keep them in this file for now